feature-libs/organization/administration/components/cost-center/details/cost-center-details.component.ts

```diff
@@ -1,19 +1,26 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
 import { CostCenter } from '@spartacus/core';
 import { Observable } from 'rxjs';
-import { shareReplay, switchMap } from 'rxjs/operators';
+import { startWith, switchMap } from 'rxjs/operators';
 import { OrganizationItemService } from '../../shared/organization-item.service';
+import { CostCenterItemService } from '../services/cost-center-item.service';
 
 @Component({
   selector: 'cx-cost-center-details',
   templateUrl: './cost-center-details.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    {
+      provide: OrganizationItemService,
+      useExisting: CostCenterItemService,
+    },
+  ],
   host: { class: 'content-wrapper' },
 })
 export class CostCenterDetailsComponent {
   model$: Observable<CostCenter> = this.itemService.key$.pipe(
     switchMap((code) => this.itemService.load(code)),
-    shareReplay({ bufferSize: 1, refCount: true })
+    startWith({})
```

Comment:
Do we need this here? I'd prefer to do this in the ...ItemService instead.

Reply:
Yes, we need it, we have tried to do this in the item service but Kris pointed out a good argument

![kris](https://user-images.githubusercontent.com/4272069/99362131-609b9c80-28b3-11eb-9202-bfae219a4915.png)

Reply:
I meant differently, basically moving the startWith to a lower layer.